tools/de-identification/ARX/README.md

```diff
@@ -8,4 +8,6 @@
 
 **GitHub User Serving as POC:** @prasser
 
+**Affiliation/Organization(s) Contributing:** TUM – Technische Universität München
```

Comment:
Thanks. Please replace by "Technical University of Munich".

On a side note: are there any guidelines on what is considered a "contributing organization"? This is an open source project and there are quite a few organizations and also companies that have contributed to the project in the past...

Reply:
Fixed! Let me look into the contributing organization issue.

Reply:
@prasser It's up to you which organizations to include here. Would you like me to add some more? Or add a link to a page of yours that lists contributors?

Reply:
@jnear It's OK for now. I'll think about it and maybe create a new PR if I find a good solution.